src/main/resources/application.yml

```diff
@@ -18,3 +20,7 @@ queueconfig:
 healthcheck:
   frequency: 1000 #milliseconds
   filename: /tmp/fwmt-adapter-healthy
+
+caseapi:
+  host: localhost
```

Comment:
Think this will need to be to 'caseapi' now, as per this https://github.com/ONSdigital/census-rm-fieldwork-adapter/pull/4#discussion_r312878233?

Reply:
yeah that would be good

Reply:
it's now: case-api